cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -269,14 +274,19 @@ public RunRecordMeta getRun(ProgramRunId programRunId) throws Exception {
    * @throws UnauthorizedException if the principal does not have access to the program
    * @throws Exception if there was some other exception performing authorization checks
    */
-  public List<RunRecord> getRuns(ProgramId programId, ProgramRunStatus programRunStatus,
-                                 long start, long end, int limit) throws Exception {
+  public List<RunRecord> getRunRecordMetas(ProgramId programId, ProgramRunStatus programRunStatus,
```

Comment:
Same here. If it is just a rename, do a refactor to update all callsite.

Reply:
Also, if you rename the method like this, the return type should be `List<RunRecordMeta>`

Reply:
Done. Also fixed the return type. It was an oversight.